### PR TITLE
added secript

### PR DESCRIPTION
**Pull request by Abbas4tech**

### CODE REVIEW

**Review by syedabdullahali — APPROVED** [review]

**Review by mujeeb100 — APPROVED** [review]

File: app.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+const addMovieBtn = document.getElementById("add-movie-btn");
+const searchBtn = document.getElementById("search-btn");
+
+const movies = [];
+
+const filtermovieHandler = () => {
```
Comment (faran4engg) [inline app.js]: camelCase ?

File: app.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  const filteredElement = document.getElementById("filter-title").value;
```
Comment (faran4engg) [inline app.js:7]: when it returns a value, can we rename this const `filteredElement` ?

File: app.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+const renderMovieElement = (filter = "") => {
+  const movieList = document.getElementById("movie-list");
+
+  if (movies.length === 0) {
+    movieList.classList.remove("visible");
+    return;
+  } else {
+    movieList.classList.add("visible");
+  }
```
Comment (faran4engg) [inline app.js:17]: ternary ?

File: app.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+  movieList.innerHTML = "";
+
+  const filtermovieElement = !filter
+    ? movies
+    : movies.filter((movie) => movie.info.title.includes(filter));
```
Comment (faran4engg) [inline app.js]: can we leverage destructuring here ?

File: app.js [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+  filtermovieElement.forEach((movie) => {
+    const movieLi = document.createElement("li");
+    const { info, ...otherkeys } = movie;
+    console.log(otherkeys);
+    // const { title: movieTitle } = info;
+    // const { getFormattedTitle } = movie;
```
Comment (faran4engg) [inline app.js]: remove log an commented code

File: app.js [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+  }
+
+  movieList.innerHTML = "";
+
+  const filtermovieElement = !filter
+    ? movies
+    : movies.filter((movie) => movie.info.title.includes(filter));
+
+  filtermovieElement.forEach((movie) => {
+    const movieLi = document.createElement("li");
```
Comment (faran4engg) [inline app.js]: movieList ?

File: app.js [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+    const { info, ...otherkeys } = movie;
+    console.log(otherkeys);
+    // const { title: movieTitle } = info;
+    // const { getFormattedTitle } = movie;
+    let editedtext = movie.getFormattedTitle() + " - ";
+    for (const otherKeys in info) {
+      if (otherKeys !== "title") {
+        editedtext = editedtext + `${otherKeys} : ${info[otherKeys]}`;
```
Comment (faran4engg) [inline app.js]: += ?

**Review by faran4engg — COMMENTED** [review]
Added few comments ! Also please fix UI issue that I have reported on discord. 
@Abbas4tech

File: app.js (hunk 6 above)
Comment (Abbas4tech) [reply app.js]: i've already used `movieList` for an `id` of `movie-list`. Hence named here like`movieLi` !! if you `movieElement` would be more appropriate then i will change it to `movieElement` ?? @faran4engg

File: app.js (hunk 1 above)
Comment (Abbas4tech) [reply app.js]: Done ! @faran4engg

File: app.js (hunk 5 above)
Comment (Abbas4tech) [reply app.js]: Done !! @faran4engg

File: app.js (hunk 7 above)
Comment (Abbas4tech) [reply app.js]: Thanks for reminding shortcuts @faran4engg . i've made this change

File: app.js (hunk 6 above)
Comment (faran4engg) [reply app.js]: > i've already used `movieList` for an `id` of `movie-list`. Hence named here like`movieLi` !! if you `movieElement` would be more appropriate then i will change it to `movieElement` ?? @faran4engg

But movieLi, this name is not making sense. Also it is blocked scope so won't be a problem.

File: app.js (hunk 3 above)
Comment (Abbas4tech) [reply app.js:17]: used shortcircuit technique @faran4engg .

File: app.js (hunk 6 above)
Comment (Abbas4tech) [reply app.js]: changed name to `movieListItem` @faran4engg

File: app.js (hunk 4 above)
Comment (Abbas4tech) [reply app.js]: used object destructuring method . @faran4engg

File: app.js [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+const addMovieHandler = () => {
+  const title = document.getElementById("title").value.toLowerCase();
+  const extraNameValue = document.getElementById("extra-name").value;
+  const extraInfoValue = document.getElementById("extra-value").value;
+
+  if (
+    title.trim() === "" ||
+    extraNameValue.trim() === "" ||
+    extraInfoValue.trim() === ""
+  ) {
```
Comment (Tanvir04khan) [inline app.js:51]: You are checking for falsey value here can you use shorthand here?

File: app.js [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+
+const filterMovieHandler = () => {
+  const filteredElement = document.getElementById("filter-title").value;
+  renderMovieElement(filteredElement);
+};
+
+const renderMovieElement = (filter = "") => {
+  const movieList = document.getElementById("movie-list");
+
+  if (movies.length === 0) {
```
Comment (Tanvir04khan) [inline app.js:14]: Checking for falsely value, can you use shorthand?

**Review by Rehmatulla-Khan — APPROVED** [review]
lgtm

**Review by mujeeb100 — APPROVED** [review]

**Review by anask601 — APPROVED** [review]

**Comment by nasir-k** [comment]
Lgtm

**Review by nasir-k — APPROVED** [review]

**Pull request merged**